### PR TITLE
Implement msb_status

### PR DESCRIPTION
**Pull request by kawindme**

### CODE REVIEW

**Review by pacjac — CHANGES_REQUESTED** [review]
Great code, I like it! Please fix return value of msb_status and consider comments regarding msb_services.

**Review by pacjac — CHANGES_REQUESTED** [review]
Great code, I like it! Please fix return value of msb_status and consider comments regarding msb_services.

File: msb/status/status.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            print("Could not parse service status loaded and enabled.")
+            continue
+        if m := _active_pattern.match(service_lines[2]):
+            (active,) = m.groups()
+        else:
+            print("Could not parse service status active.")
+            continue
+
+        service_status[name] = {
+            "description": desc,
```
[the remark this answers is not in the document]
Comment (kawindme) [reply msb/status/status.py:107]: Does it "hurt" to send it along? If we omit it, the description would need to be added manually to the dashboard.
I don't have a strong opinion on this, if you confirm I will remove it.

File: msb/status/status.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+
+_name_desc_pattern = re.compile(r"^(msb-\w*\.service)\s-\s([\w\s]*)")
+_loaded_enabled_pattern = re.compile(
+    r"^Loaded:\s(loaded)\s\(/etc/systemd/system/msb-\w+\.service;\s(\w*);"
+)
+_active_pattern = re.compile(r"^Active:\s(\w*\s\(\w*\))")
+
+
+def msb_services() -> dict:
```
[the remark this answers is not in the document]
Comment (kawindme) [reply msb/status/status.py:80]: I guess I choose bad names for these properties as they actually are not boolean. For example "active" corresponds to the "ActiveState" as described in the man pages, see e.g. https://www.man7.org/linux/man-pages/man5/org.freedesktop.systemd1.5.html . 
Reading the man-page I also found that using `systemctl status` is not recommended way to read the status, programmatically. Instead, `systemctl show` should be used.
I will rework this part using `show`.

**Comment by kawindme** [comment]
I reimplemented the msb_services function. Please review again.

**Review by pacjac — APPROVED** [review]

**Pull request merged**